.travis.yml

```diff
@@ -1,13 +1,22 @@
 language: python
+
 sudo: false
+
 services: cassandra
 
 env:
-  - CASS_DRIVER_NO_CYTHON=1
+  matrix:
+    - TOX_ENV=py27-django110
```

Comment:
Django 1.8+ should work, the issue is that the `model_meta` tests will only work with django 1.10 (i think). You could skip that test folder for those version maybe? Otherwise would have to have model_meta tests for each version (1.8,1.9,1.10) as there's some differences.